### ISSUE TITLE
Hints for nested properties in Sensor `payload_schema` (house.dog.fleas)

### ISSUE THREAD

**Issue by armab** [StackStorm/st2web#219]
The cool thing I found (suddenly :facepunch:) is that Sensor payload schema supports `description` for each property and even more, webui shows the hints!
![1](https://cloud.githubusercontent.com/assets/1533818/9892230/bfdc3f0a-5c04-11e5-875c-43fdbce8be2c.gif)
That's just awesome, however never seen someone uses that in any docs or examples.

----

The cool thing will be also to show hints for every child property in sensor `payload_schema`, like `house.dog.fleas`

**Comment by armab** [comment]
Oh, I could find related discussion: https://github.com/StackStorm/st2web/pull/191
Is something like that already implemented?

**Comment by emedvedev** [comment]
Sadly, no. Before the PR you've mentioned, `house.dog.flies` didn't even validate to begin with, let alone description, so at least we fixed that. :)
As far as I know (I'm not sure, so @enykeev can correct me on that), we don't even get data about child properties. So that's not even a web UI issue: we suggest everything that comes via API.

**Comment by enykeev** [comment]
Yes and no. We got all the data we need to suggest\validate the criteria. The problems are a) users rarely add the info for the second level of payload so b) we don't make an additional effort to parse `parameters` within another `type: object`. As soon as it will become something people actually need, there's not too much effort required to add it, but now we're spread thin and have to prioritize.

**Comment by armab** [comment]
Yeah, as I checked, data about the nested properties is returned from API. So it's only about `st2web` implementation.

Majority of data returned from Sensors is structurized and nested this way, 90%+ of Sensors written by us (or by @Kami :)
I believe if we'll start documenting all future community Sensors as much as it possible for better autocomplete support in WebUI - better it will be for users and easier to use the product.

At least I'll start documenting new sensors this way from now (where it makes sense), hoping that it will be supported in `webui` someday :calendar: .

I agree that such cosmetic change is not a priority, let's keep this feature request opened :zzz:
